README.md

```diff
@@ -28,6 +28,17 @@
 
 # Triton Inference Server
 
+📣 **vLLM x Triton Meetup at Fort Mason on Sept 9th 4:00 - 6:30 pm**
```

Comment:
One spot says 4-6:30pm, and another says 4-9pm. Should they match?

Reply:
yes thanks for catching!